modules/utilities.json

```diff
@@ -2,16 +2,50 @@
   "title": "Utilities",
   "modules": {
     "level-iterator-stream": {
-      "github": "level/iterator-stream",
-      "description": "Turn an [abstract-leveldown] iterator into a readable stream."
+      "github": "Level/iterator-stream",
+      "description": "Turn an [abstract-leveldown] iterator into a readable stream. Included in [levelup].",
+      "copyrightYear": 2012,
+      "author": null
+    },
+    "level-concat-iterator": {
+      "github": "Level/concat-iterator",
+      "description": "Concatenate items from an iterator into an array.",
+      "copyrightYear": 2018,
+      "author": null
+    },
+    "level-codec": {
+      "github": "Level/codec",
+      "description": "Encode keys, values and ltgte options. Used in [encoding-down].",
+      "copyrightYear": 2012,
+      "author": null
+    },
+    "level-errors": {
+      "github": "Level/errors",
+      "description": "Error types for [levelup].",
+      "copyrightYear": 2012,
+      "author": null
     },
     "level-test": {
-      "github": "level/level-test",
-      "description": "Easily run your `level-*` tests against [all stores](#stores)."
+      "github": "Level/level-test",
+      "description": "Easily run your `level-*` tests against [all stores](#stores).",
+      "copyrightYear": 2013,
+      "author": "Dominic Tarr"
     },
     "lev2": {
       "github": "maxlath/lev2",
       "description": "A complete REPL & CLI for managing LevelDB instances"
+    },
+    "level-ws": {
+      "github": "Level/level-ws",
+      "description": "A basic WriteStream implementation for [levelup].",
+      "copyrightYear": 2012,
+      "author": null
+    },
+    "level-lazy-open": {
+      "github": "Level/lazy-open",
+      "description": "Lazily open a leveldown compatible backend. Superseded by [deferred-leveldown].",
```

Comment:
@ralphtheninja I have never used or looked into `level-lazy-open`. Am I right that it is superseded by `deferred-leveldown` (if so, perhaps we should deprecate `level-lazy-open`)?

Reply:
Nevermind, I'll remove this note here and open an issue in `lazy-open` instead.

Reply:
=> https://github.com/Level/lazy-open/issues/42